### PR TITLE
Terraform realtime etl

### LINKED ISSUE #78: Terraform Report pipeline
### Task

- [x] Terraform Lambda for extracting from RDS and transforming station data
- [x] Terraform Event bridge to call Lambda everyday at 12am 

### User Story
As a developer, I want to automate the process of generating a PDF report on AWS so that I can maintain and repeat this process if need be.

### LINKED ISSUE #89: Terraform RealTimesTrains Pipeline
### Task

- [x] Create lambda with terraform to run the ETL pipeline.
- [x] Create an event bridge schedule with terraform to schedule the lambda to run everyday at 12am.

### User Story
As a developer, I want to automate the process of creating the performance pipeline lambda on AWS and scheduling it to run so that I can maintain and repeat this process if need be.

### PR DESCRIPTION
**Pull request by FarihaChoudhury**
Terraform for lambda to run Realtime pipeline.
Event bridge schedules lambda to run every 12am.

closes #89

### CODE REVIEW

File: realtime_trains/README.md [hunk 1]
```diff
@@ -31,8 +31,13 @@ This will prompt you to enter a station name or code. After providing the input,
 Before running the script, you need to set up your National Rail API credentials. Create a new file called `.env` in the `realtime_trains` directory and add the following lines, replacing `YOUR_USERNAME` and `YOUR_PASSWORD` with your actual username and password:
```
Comment (A-Faris) [inline realtime_trains/README.md]:
```suggestion
Before running the script, you need to set up your National Rail API credentials. Create a new file called `.env` in the `realtime_trains` directory and add the following lines, replacing `your_username` and `your_password` with your actual username and password:
```

**Review by A-Faris — DISMISSED** [review]
Everything looks great, just made one change to be constant with your edit

**Review by A-Faris — DISMISSED** [review]
That's good

**Review by Zhi-704 — APPROVED** [review]
Looks good!

**Pull request merged**